pages/profile/community/index.js

```diff
@@ -26,13 +31,11 @@ export default function Community({ user, allUsers, mutateAllUsers }) {
         <ProfileCard
           key={communityUser._id}
           foundUser={communityUser}
-          isFriend={user.friends.includes(communityUser._id)}
-          isRequested={communityUser.connectionRequests.some(
-            (request) => request.senderId === user._id
-          )}
           user={user}
           allUsers={allUsers}
           mutateAllUsers={mutateAllUsers}
+          mutateUser={mutateUser}
+          followButton={true}
```

Comment:
Possible shorthand for a prop whose value is true. 

```suggestion
          followButton
```